test/stage2/test.zig

```diff
@@ -295,7 +256,7 @@ pub fn addCases(ctx: *TestContext) !void {
             \\    );
             \\    unreachable;
             \\}
-        ,
+            ,
```

Comment:
`zig fmt` changed this. Will there be any conflicts with other PRs?

Reply:
I just merged #6293 which I believe reverts this, mind running it again with the latest master?

Reply:
Will do. Just a sec...

Reply:
Done.